src/test/java/org/symphonyoss/symphony/messageml/MessageMLContextTest.java

```diff
@@ -110,6 +111,40 @@ public void testParseMessageML() throws Exception {
     assertEquals("Legacy entities", MAPPER.writeValueAsString(expectedEntities), MAPPER.writeValueAsString(entities));
   }
 
+  @Test
```

Comment:
These Junit are only a draft used for development. TODO: modify the main JUnit with elements + labels

Reply:
full JUnit added